debian/control

```diff
@@ -0,0 +1,31 @@
+Source: tvb-library
+Maintainer: the virtual brain <tvb.debian@codemart.ro>
+Uploaders: umar haruna abdullahi <umarbrowser20@gmail.com>
+Section: python
+Priority: optional
+Build-Depends: debhelper (>=11),
+               dh-python,
+               python-all,
+               python-setuptools,
+               python-networkx,
+               python-numpy,
+               python-numba,
+               python-numexpr,
+               python-matplotlib,
+               python-sklearn,
+               python-scipy,
+               tvb-gdist,
+               tvb-data
```

Comment:
move to use python3-* packages

Reply:
remove tvb-data from dependencies

Reply:
add pytest to be able to run unittests at package build time
verify that they do run at package build time, if not, adjust debian/rules accordingly with `override_dh_auto_test:` rule which would trigger/run the tests